common/determined_common/experimental/determined.py

```diff
@@ -23,23 +29,43 @@ class Determined:
     def __init__(
         self,
         master: Optional[str] = None,
-        user: Optional[str] = None,
+        user: Optional[str] = "determined",
+        password: Optional[str] = "",
```

Comment:
We're going to need an alternative to this so that you can write working code and push it to GitHub without exposing credentials. I'm a fan of how AWS does credentials.